app/templates/application.hbs

```diff
@@ -1,5 +1,11 @@
-{{#if session.isLogged}}
-  {{partial "partials/logged"}}
+{{#if session.isExpired}}
+  <div class="session-error">
```

Comment:
mmm for me it's to much, creating another partial only for show a warning... but, in the end it's an opinion, you think that should be moved? try to convince me ;)